linux/step04_all_omero_install.sh

```diff
@@ -8,9 +8,11 @@ set -eux
 
 #start-install
 if [ $OMEROVER == "latest" ]; then
+	# version number and ice version to be dropped
 	#start-release-ice36
 	cd /opt/omero/server
-	SERVER=https://downloads.openmicroscopy.org/omero/5.6/server-ice36.zip
+	SERVER=https://github.com/ome/openmicroscopy/releases/latest/download/OMERO.server-5.6.13-ice36.zip
```

Comment:
``OMERO.server-5.6.13-ice36.zip`` seems hardcoded ? Maybe I misunderstand, what will happen once we have OMERO.server-5.6.14 ?

Reply:
The PR https://github.com/ome/openmicroscopy/pull/6414 will drop the version 
and push OMERO.server.zip
This means that we won't need to upgrade it every time